packages/firestore/src/local/local_store.ts

```diff
@@ -909,7 +915,8 @@ class LocalStoreImpl implements LocalStore {
                   target,
                   targetId,
                   TargetPurpose.Listen,
-                  txn.currentSequenceNumber
+                  txn.currentSequenceNumber,
+                  readFrom || SnapshotVersion.min()
```

Comment:
What happens when we don't go into this branch? Should we not still update the target data, even if we already have a (potentially outdated) local copy?

While I am here, is "targetData.snapshotVersion" the same as "readTime"? Do we need both?

Reply:
Added code to update local copy. And removed `|| SnapshotVersion.min()`(that is what you meant right?).